src/sentry/notifications/notifications/rules.py

```diff
@@ -145,6 +147,13 @@ def get_context(self) -> MutableMapping[str, Any]:
                 },
             )
 
+        if features.has("organizations:mute-alerts", self.organization) and len(self.rules) > 0:
+            context["snooze_alert"] = True
+            context["snooze_alert_url"] = (
+                f"/organizations/{self.organization.slug}/alerts/rules/{self.project.slug}/{self.rules[0].id}/details/"
+                + urlencode({"mute": "1"})
```

Comment:
@roggenkemper we probably should use `get_sentry_query_params` to make sure we get all the query params we need.